test/manual/basic.html

```diff
@@ -7,7 +7,10 @@
 
 An alert should be shown with the <code>{"name":"axios"}</code>
 
-<script src="promise.js"></script>
+<script>
+if (typeof Promise !== "function")
+    document.write('<script src="//cdnjs.cloudflare.com/ajax/libs/es6-promise/4.1.1/es6-promise.min.js"><\/script>');
+</script>
```

Comment:
> Added a Cloudflare CDN polyfill for promise, which should fix the IE build breaking in continuous integration in master builds.

I recalled a question here. What's the different between those two promises? And did you really test by IE?

Reply:
@chinesedfan  sorry for the late reply. The Promise here is only really needed for running the unit tests in IE.

In my opinion, the unit tests can/should be moved to a brower-less testing approach, which would allow for the removal of any `Promise` polyfill and faster testing, etc.

Reply:
@chinesedfan I had some trouble getting the IE VM from Microsoft up and running. Maybe Github actions can help with the problem here. They explicitly support the Windows containers:

https://github.com/features/actions